Source/QED/QuantumSyncEngineWrapper.h

```diff
@@ -0,0 +1,44 @@
+#ifndef WARPX_quantum_sync_engine_wrapper_h_
+#define WARPX_quantum_sync_engine_wrapper_h_
+
+#include "QedWrapperCommons.h"
+
+//QS ENGINE from PICSAR
+#include "quantum_sync_engine.hpp"
+
+using WarpXQuantumSynchrotronWrapper =
+    picsar::multi_physics::quantum_synchrotron_engine<amrex::Real, DummyStruct>;
+
+// Functors ==================================
+
+// These functors provide the core elementary functions of the library
+// Can be included in GPU kernels
+
+/* \brief Functor to initialize the optical depth of leptons for the
+*   Quantum Synchrotron process */
+class QuantumSynchrotronGetOpticalDepth
+{
+public:
+    QuantumSynchrotronGetOpticalDepth ()
+    {};
+
+    AMREX_GPU_DEVICE
+    amrex::Real operator() () const;
```

Comment:
Same thing, shouldn't this operator be inlined?

Reply:
You are right. I've addressed this point as above.